en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,579 @@
+
+## Prerequisites
+
+Before deploying a TiDB cluster on Azure AKS, make sure the following requirements are satisfied:
+
+* Install [Helm 3](https://helm.sh/docs/intro/install/): used for deploying TiDB Operator.
+* Complete all operations in [Create a Kubernetes cluster](https://docs.microsoft.com/en-us/azure/aks/tutorial-kubernetes-deploy-cluster) and [Create a new cluster that can use Ultra disks](https://docs.microsoft.com/en-us/azure/aks/use-ultra-disks).
+
+    This guide includes the following contents:
+
+    * Install and configure `az cli` used for creating Kubernetes clusters.
+    * Create the `AKS cluster` with Ultra disks enabled, or enable Ultra disks on an existing cluster.
+
+To verify whether AZ CLI is configured correctly, run the `az login` command. If login with account credentials successfully, AZ CLI is configured correctly. Otherwise, you need to re-configure AZ CLI.
+
+> **Note:**
+>
+> The operations described in this document requires these permissions [AKS service permissions](https://docs.microsoft.com/en-us/azure/aks/concepts-identity#aks-service-permissions).
+> If the cluster's kubernetes version < 1.21 you need install **aks-preview CLI extension** for using Ultra Disks, and need **EnableAzureDiskFileCSIDriver** registered in your subscription
+
+The following command will install [aks-preview CLI extension](https://docs.microsoft.com/en-us/azure/aks/custom-node-configuration#install-aks-preview-cli-extension)
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+az extension add --name aks-preview
+```
+
+The following command will register [EnableAzureDiskFileCSIDriver](https://docs.microsoft.com/en-us/azure/aks/csi-storage-drivers#install-csi-storage-drivers-on-a-new-cluster-with-version--121) in [your subscription](https://docs.microsoft.com/en-us/cli/azure/feature?view=azure-cli-latest#az_feature_register-optional-parameters)
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+az feature register --name EnableAzureDiskFileCSIDriver --namespace Microsoft.ContainerService --subscription ${your-subscription-id}
+```
+
+## Create a AKS cluster and node pools
+
+most of the TiDB cluster components use Azure disk as storage, according to AKS [Best Practice Document](https://docs.microsoft.com/en-us/azure/aks/operator-best-practices-cluster-isolation), it is recommended to create a node pool with every availability zone (at least 3 in total) for each component when creating an AKS cluster.
+
+### create AKS cluster with [CSI enabled](https://docs.microsoft.com/en-us/azure/aks/csi-storage-drivers)
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+# create AKS cluster
+az aks create \
+    --resource-group ${resourceGroup} \
+    --name ${clusterName} \
+    --location ${location} \
+    --generate-ssh-keys \
+    --vm-set-type VirtualMachineScaleSets \
+    --load-balancer-sku standard \
+    --node-count 3 \
+    --zones 1 2 3 \
+    --aks-custom-headers EnableAzureDiskFileCSIDriver=true
```

Comment:
This parameter don't need when Kubernetes version >= 1.21

Reply:
Thank you,  have added note in 'cluster create' step